aten/src/ATen/core/common.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
```

Comment:
we already have ATenGeneral.h  / Macros.h for common stuff that is included, I don't think we need another file.  Maybe just move this to ATenGeneral.h?